addons/mail/tests/test_mail_thread.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.mail.tests.common import MailCommon
+from odoo.tests import tagged
+
+
+@tagged('mail_thread', 'mail')
+class TestMailThread(MailCommon):
+
+    def test_search_message_partner_ids(self):
+        '''
+            The purpose of this test case is to check that _search_message_partner_ids does
+            not raise a traceback when called on a false domain
+        '''
```

Comment:
Most tests should be in `test_mail` unless it needs to be in mail tests for some reason, or if it's very technical and already has a test class there.
 
There's already a test `test_field_message_partner_ids` in `test_mail`, would add a test under it.